### ISSUE TITLE
Outdated Feature Overview Demo

### ISSUE THREAD

**Issue by Microwave-WYB** [wbkd/react-flow#2377]
### Describe the Bug

The demo code on https://reactflow.dev/docs/examples/overview/ does not run correctly on the latest version because of the missing width and height

### Steps to Reproduce the Bug or Issue

1. Initialize project with `npm init react-app <app_name>`
2. Copy the code for `App.js` and `initial-elements.js` from https://reactflow.dev/docs/examples/overview/
3. `npm start`

### Expected behavior

What's shown in the feature overview

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/68882116/185913315-b4588724-5e31-43eb-9887-e929edc403f8.png)
The app worked after  adding the height
![image](https://user-images.githubusercontent.com/68882116/185913436-3f98f7d1-bb00-4e55-bcc2-cb9774b53dec.png)

### Platform

- OS: Linux
- Browser: Chrome/Firefox
- Version: Chrome 103.0.5060.53 , Firefox 103.0.2

**Comment by moklick** [comment]
![Screenshot 2022-08-22 at 13 54 05](https://user-images.githubusercontent.com/2857535/185915227-7f76bfe5-2ed5-42db-9b3e-74118232601d.png)

There is a little button "Open in Codesandbox" in the bottom right corner that shows the whole code of that example. But you have a good point here. It's not very beginner friendly. We will keep this open for now.

**Comment by raarts** [comment]
Adding to this: a page on coordinates (both pixel as well as react-flow) and width/height and how to retrieve them.
Both when inside a custom node as well a in a ReactFlow handler. I have spent a lot of time already with react-flow, and this keeps stopping me in my tracks.

(latest hurdles for me: 
- inside a custom node draghandler the mouse coordinates are easy to get, but getting the reactflow outer bounds is hard from there, even though width+height are in the store.
- getting the coordinates of the custom node or width/height in pixels is challenging too).

A page dedicated to this subject with code examples would be very helpful for beginners.

**Comment by bcakmakoglu** [comment]
@raarts FYI: Custom Nodes receive `xPos` and `yPos` as props so you should be able to fetch the coordinates from there pretty easily :)

```ts
// props that get passed to a custom node
export interface NodeProps<T = any> {
  id: string;
  type: string;
  data: T;
  selected: boolean;
  isConnectable: boolean;
  xPos: number;
  yPos: number;
  dragging: boolean;
  zIndex: number;
  targetPosition?: Position;
  sourcePosition?: Position;
  dragHandle?: string;
}
```

**Comment by raarts** [comment]
@bcakmakoglu  Yes, I know, but I needed it in browser pixels so I gathered I needed something like a reverse project() function. There isn't one, so I ended up just copying it out of the reactflow sourcecode, and reverse it myself.
